opre_ops/ops_site/models.py

```diff
@@ -28,5 +31,43 @@ class Meta:
     def display_name(self):
         return self.first_name + " " + self.last_name
     display_name.short_description = "Full name"
-    
     full_name = property(display_name)
+
+    def __str__(self):
+        return self.full_name
+
+
+ARRANGEMENT_TYPES = [("1", "OPRE Appropriation"), ("2", "Cost Share"), ("3", "IAA"), 
+                    ("4", "IDDA"), ("5", "MOU")]
+class CANInfo(models.Model):
```

Comment:
[_suggestion_]: A docstring comment here explaining what a CAN is, plus any other important information that a dev working on this model should know about the CANInfo model.